src/routes/Header.js

```diff
@@ -0,0 +1,69 @@
+/* eslint-disable jsx-a11y/alt-text */
+import React from 'react';
+import SearchIcon from '@mui/icons-material/Search';
+import ShoppingCartIcon from '@mui/icons-material/ShoppingCart';
+import LocalPhoneIcon from '@mui/icons-material/LocalPhone';
+
+function Header() {
+  return (
+    <div className='w-full'>
+      <div>
+        <div className='bg-black flex justify-end h-10 items-center'>
+          <div className='mr-10 flex'>
+            <button className='mr-20 h-10 text-white hover:text-gray-400 hover:bg-gray-300'>Tài khoản</button>
+
+            <button className='mr-20 h-10 text-white hover:text-gray-400 hover:bg-gray-300'>Giỏ hàng</button>
+
+            <button className='mr-20 h-10 text-white hover:text-gray-400 hover:bg-gray-300'>Đăng nhập</button>
+          </div>
+        </div>
+      </div>
+
+      <div className='bg-[#D2ECED] h-56 flex justify-center '>
+        <div className='w-1600 items-start justify-between flex'>
+        <img className='h-56' src='./images/logo.png' alt='not-found' />
+        
+
+          
+            <div className='flex mt-20 ml-16'>
+              <input className='w-700 h-14'></input>
+              <button className=' bg-blue-300 h-14 w-14 hover:text-blue-900 justify-center'>
+                <SearchIcon />
+              </button>
+            
+          </div>
+          <div className='flex ml-64 mt-20'>
+            <div className='w-14 h-14 bg-white flex items-center justify-center'>
+              <LocalPhoneIcon />
+            </div>
+            <div className='w-40 h-14 bg-blue-300'>
+              <div>
+                <p className='flex justify-center'>CALL ME US</p>
+                <p className='flex justify-center'>079-558-2457</p>
+              </div>
+            </div>
+          </div>
+        </div>
+      </div>
+
+      <div>
+        <div className='bg-black flex justify-start h-16 items-center'>
+          <div className='w-full flex'>
+            <button className='w-40 h-16 text-white hover:text-gray-400 hover:bg-gray-300'>TRANG CHỦ</button>
+            <button className='w-40 h-16 text-white hover:text-gray-400 hover:bg-gray-300'>NƯỚC HOA NAM</button>
+            <button className='w-40 h-16 text-white hover:text-gray-400 hover:bg-gray-300'>NƯỚC HOA NỮ</button>
+            <button className='w-40 h-16 text-white hover:text-gray-400 hover:bg-gray-300'>LIÊN HỆ</button>
+          </div>
+          <div className='flex justify-center mr-40'>
+            <button className='bg-blue-300 w-16 h-16 hover:text-white'>

```

Comment:
button add to cart là 1 cái  button chứa 2 thẻ div bên trong chứ k phải 2 button riêng lẻ nhé